Tests/test_image_getdominantcolors.py

```diff
@@ -0,0 +1,18 @@
+from .helper import hopper
+
+
+def test_getdominantcolors():
+    def getdominantcolors(mode):
+        im = hopper(mode)
+        colors = im.getdominantcolors()
+        return len(colors)
+
+    assert getdominantcolors("F") == 3
+    assert getdominantcolors("I") == 3
+    assert getdominantcolors("L") == 3
+    assert getdominantcolors("P") == 3
+    assert getdominantcolors("RGB") == 3
+    assert getdominantcolors("YCbCr") == 3
+    assert getdominantcolors("CMYK") == 3
+    assert getdominantcolors("RGBA") == 3
+    assert getdominantcolors("HSV") == 3
```

Comment:
We can improve these tests so they're testing more than just they return a thing of length three.

We can also use `@pytest.mark.parametrize` so each test case is run independently so cannot depend on an earlier case.

Here's an example:

```python
import pytest

from .helper import hopper


@pytest.mark.parametrize(
    "test_mode, expected",
    [
        ("F", [28.8104386146252, 66.26185757773263, 127.53211228743844]),
        ("I", [136, 94, 40]),
        ("L", [25, 63, 127]),
        ("P", [11, 71, 159]),
        ("RGB", [(172, 117, 94), (53, 44, 55), (95, 127, 185)]),
        ("YCbCr", [(130, 108, 155), (123, 163, 105), (47, 131, 131)]),
        ("CMYK", [(201, 210, 199, 0), (159, 127, 69, 0), (82, 137, 160, 0)]),
        ("RGBA", [(31, 24, 37, 255), (129, 125, 150, 255), (87, 67, 72, 255)]),
        ("HSV", [(140, 131, 85), (177, 70, 46), (97, 131, 186)]),
    ],
)
def test_getdominantcolors(test_mode, expected):
    def getdominantcolors(mode):
        im = hopper(mode)
        im.thumbnail((10, 10))
        colors = im.getdominantcolors()
        return colors

    assert getdominantcolors(test_mode) == expected
```

What do you think?

Do those expected return values look right?

Might they be different on other systems? Especially the first one. If so, we can adjust it somewhat, possibly have a special case for that.

It would be good to add further `test_X` functions to verify the other parameters of `im.getdominantcolors`, plus warnings and exceptions.